Sources/Identity/CustomerInfoManager.swift

```diff
@@ -90,27 +90,67 @@ class CustomerInfoManager {
         sendUpdateIfChanged(customerInfo: info)
     }
 
-    func customerInfo(appUserID: String, completion: ((Result<CustomerInfo, BackendError>) -> Void)?) {
-        let infoFromCache = cachedCustomerInfo(appUserID: appUserID)
-        var completionCalled = false
+    func customerInfo(
+        appUserID: String,
+        fetchPolicy: CacheFetchPolicy,
+        completion: ((Result<CustomerInfo, BackendError>) -> Void)?
+    ) {
+        switch fetchPolicy {
+        case .fromCacheOnly:
```

Comment:
I'm working on the android side of this, and one thing I'm wondering is, do we want to fetch new data even with this fetch policy (if stale)? I think that might be better so we update as early as possible, but I could also understand devs thinking that the cache will not be updated in this case... Though if that's the case we should make sure we handle all other possible places where the cache can be updated. cc @NachoSoto @aboedo

Reply:
The [docs for CachePolicy](https://revenuecat.github.io/purchases-ios-docs/4.6.1/documentation/revenuecat/cachefetchpolicy) might help illustrate the desired behavior here. 
For this specific case, if there's nothing in cache, we'd return an error right away no fetching. If there's something in cache, it gets returned even if stale. FWIW I think most apps will not want to use this behavior, and will opt for any of the others

Reply:
Happy to clarify further if needed!

Reply:
I agree that this won't be used much by devs so it's not that big of a deal. The docs don't specify whether this would initiate a fetch, and it doesn't either for some other policies, like the `cacheOrFetched` policy, it says: `Default behavior: returns the cached data if available (even if stale), or fetches up-to-date data.`, it says `or` but it can be an `and` if the cache is stale (I'm probably nitpicking here).

For now I will follow iOS behavior on this, but I'm more concerned about this other question: https://github.com/RevenueCat/purchases-ios/pull/1608#discussion_r895928743 @aboedo

Reply:
That is certainly valid feedback. We should update the docs to make it 100% unambiguous, i.e.: specify for each one: 
- If cache is available and valid, ...
- If cache is available and stale, ... 
- If cache is not available, ...
Where the second part specifies whether an error or value is returned, whether it'll trigger a fetch, and whether it'll wait for the fetch before returning or it'll trigger something in the delegate instead.

Maybe we could specify that in the Android implementation and copy the text back to this one?